frontend/src/scenes/max/maxLogic.ts

```diff
@@ -318,6 +318,20 @@ export const maxLogic = kea<maxLogicType>([
                 return threadGrouped
             },
         ],
+        formPending: [
+            (s) => [s.threadRaw],
+            (threadRaw) => {
+                const lastMessage = threadRaw[threadRaw.length - 1]
+                if (lastMessage && isAssistantMessage(lastMessage)) {
+                    return !!lastMessage.meta?.form
+                }
+                return false
+            },
+        ],
+        inputDisabled: [
```

Comment:
Would be smoother UX if we turned this into `inputDisabledReason`, so that in the form pending case we could show something like "Please choose one of the proposed answers"

Reply:
Ah, since this is on the text area, which doesn't have `disabledReason`, let's skip